instruqt/support-cli/03-collect-support-bundle/solve-kubernetes-vm

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
Should we remove this file?

Reply:
Or better, it will need to actually solve the challenge. Else the `check` lifecycle script will not work for `instruqt track test`

Reply:
yes, should remove it, not needed in the end.  Probably better to not have a solve script as it's the point of the lab to debug, there is always the Exit option ;)

Reply:
though, do we want to have the solve on all the lab challenges generally?  Haven't looked at the track test functionality, maybe as a rework later to skim over them all?

Reply:
At some point yes. And for some tracks we're already doing it. Adding a solve script can help users get unstuck without us having to help

Reply:
I think the solve script needs to be present to allow a skip, even if there isn't anything to "solve" from that step.